.github/workflows/alerts.yaml

```
@@ -28,5 +28,5 @@ jobs:
           alert_file_path: monitoring/alerts.yaml
           test_file_path: ${{ matrix.tests.file }}
           alert_inputs: >-
-            namespace=zenko,service=artesca-data-connector-s3api-metrics,replicas=3
+            namespace=zenko,service=artesca-data-connector-s3api-metrics,reportJob=artesca-data-ops-report-handler,replicas=3
```

should be put on multiple lines, to improve readability (c.f. https://github.com/scality/backbeat/blob/development/8.6/.github/workflows/alerts.yaml#L18-L34)

```suggestion
          alert_inputs: |
            namespace=zenko
            service=artesca-data-connector-s3api-metrics
            reportJob=artesca-data-ops-report-handler
            replicas=3
```

(this requires updted `scality/action-prom-render-test@1.0.3`, but this should be already the case on tip of dev branches: did you rebase on latest changes?)

---

yes I am up to date, so I will bump the version also

---

Done and passing, I bumped the action version and applied your suggestions

---

I must have missed it when making the ghcr migration...

can you also bump `checkout@v4` ?

---

Sure, let me add a commit

---

Done